### PR TITLE
AC Test Fix - LRAC-10962 | master

### PR DESCRIPTION
**Pull request by ViniLopes20**
https://issues.liferay.com/browse/LRAC-10962
[Testray](https://testray.liferay.com/home/-/testray/case_results/index?p_p_state=normal&orderByCol=status_sortable&orderByType=asc&testrayBuildId=1664799873)

### CODE REVIEW

**Comment by ViniLopes20** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PageEvents.testcase [hunk 1]
```diff
@@ -7,10 +7,6 @@ definition {
 	property testray.main.component.name = "Analytics Cloud";
 
 	setUp {
-		AntCommand(
-			locator1 = "build-test.xml",
-			value1 = "record-test-generated-properties");
-
```
Comment (manuferreira) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PageEvents.testcase:13]: hey Vini, can you explain why did you remove these lines?

Comment (ViniLopes20) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PageEvents.testcase:13]: Hey @manuferreira, when Afonso fixed a part of these event tests he had put this function at the beginning of the setUp to be executed in each test, but when I noticed that several event test files don't have this macro and when I made other corrections this macro did not change the result when it was executed or not. So since it doesn't change anything on the test, I removed it.

**Review by joshitagaki — APPROVED** [review]
LGTM!

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PageEvents.testcase (hunk 1 above)
Comment (manuferreira) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PageEvents.testcase:13]: Ohh, I remember now. Thanks for the clarification. :+1:

**Review by manuferreira — APPROVED** [review]
LGTM!

**Review by christianstokes — APPROVED** [review]

**Comment by christianstokes** [comment]
ci:forward
